### PR TITLE
go-feature-flag: Add version 1.14.1

### PR DESCRIPTION
**Pull request by thomaspoignant**
Adding GO Feature Flag inside the scoop extra bucket.

fixes https://github.com/thomaspoignant/go-feature-flag/issues/817

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by HUMORCE** [comment]
I noticed that it does not included utils(lint, migration-cli, relay-proxy). If this are not important, then I will merge this PR later.

**Review by HUMORCE — APPROVED** [review]

**Comment by thomaspoignant** [comment]
> I noticed that it does not included utils(lint, migration-cli, relay-proxy). If this are not important, then I will merge this PR later.

Yes we can add `lint` and `migration-cli` too but not sure if this should be in the same PR or it should be in another PR?

**Comment by HUMORCE** [comment]
They can be downloaded and installed in the same(this) manifest.

**Comment by thomaspoignant** [comment]
> They can be downloaded and installed in the same(this) manifest.

Nice, I will update this PR.

**Comment by thomaspoignant** [comment]
@HUMORCE I have added the `migration-cli` and the `lint`, relay-proxy is out of scope since we are deprecating the usage of this package name.

File: bucket/go-feature-flag.json [hunk 1]
```diff
@@ -5,16 +5,40 @@
...
+                "https://github.com/thomaspoignant/go-feature-flag/releases/download/v1.14.1/go-feature-flag-lint_1.14.1_Windows_arm64.tar.gz"
+            ],
+            "hash": [
+                "685705e0b619f11627ababef107ac2f596999b6277cbad7238027fc7a1ca3fe3",
+                "022c2cf68d3808e7e687035854918c5d9faefb70c8592b75b037e37481b37de7",
+                "934e5588a1761a5e2d2da990a91aadee1d9c4f50bf33e5896c3ad13992cbae3e"
+            ]
         }
     },
     "bin": "go-feature-flag.exe",
```
Comment (HUMORCE) [inline bucket/go-feature-flag.json:44]: shims

**Review by HUMORCE — CHANGES_REQUESTED** [review]

File: bucket/go-feature-flag.json (hunk 1 above)
Comment (thomaspoignant) [reply bucket/go-feature-flag.json]: fixed !

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**